examples/js/datalayer.mocks.1.js

```diff
@@ -447,12 +438,11 @@ governing permissions and limitations under the License.
       adobeDataLayer.push({
         event: 'clicked',
         data: {
-          invalid: {}
```

Comment:
this test does not make sense anymore as there is no invalid format: I'd remove it.